uber/tests/models/test_attendee.py

```diff
@@ -251,11 +266,30 @@ def unset_volunteering(self, monkeypatch):
         return Attendee.unset_volunteering
 
     def test_dept_head_invariants(self):
-        a = Attendee(ribbon=c.DEPT_HEAD_RIBBON)
+        a = Attendee(ribbon=c.DEPT_HEAD_RIBBON, assigned_depts=c.CONSOLE)
         a._staffing_adjustments()
-        assert a.staffing and a.trusted
+        assert a.staffing
+        assert a.trusted_in(c.CONSOLE)
+        assert a.trusted_in_any_depts
         assert a.badge_type == c.STAFF_BADGE
 
+    def test_trust_TrustedInAssignedDept_AfterStaffingAdjustments(self):
+        a = Attendee(staffing=True,
+                     assigned_depts='{},{}'.format(c.CONSOLE, c.CON_OPS),
+                     trusted_depts='{},{}'.format(c.CONSOLE, c.CON_OPS))
+        a._staffing_adjustments()
+        assert a.assigned_to(c.CONSOLE) and a.trusted_in(c.CONSOLE)
+        assert a.assigned_to(c.CON_OPS) and a.trusted_in(c.CON_OPS)
+
+    def test_trust_NotTrustedInAnUnassignedDept_AfterStaffingAdjustments(self):
+        a = Attendee(staffing=True,
+                     assigned_depts='{},{}'.format(c.CONSOLE, c.CON_OPS),
+                     trusted_depts='{},{}'.format(c.ARCADE, c.CON_OPS))
+        a._staffing_adjustments()
+        assert a.assigned_to(c.CONSOLE) and not a.trusted_in(c.CONSOLE)
+        assert not a.assigned_to(c.ARCADE) and not a.trusted_in(c.ARCADE)
+        assert a.assigned_to(c.CON_OPS) and a.trusted_in(c.CON_OPS)
+
```

Comment:
oh man yea that works great, I changed the names back to be less verbose and added docstrings.

example output from running the tests in PyCharm when they fail:

```python
.F
self = <uber.tests.models.test_attendee.TestStaffingAdjustments object at 0x7f1029b36cf8>

    def test_staffing_still_trusted_assigned(self):
        """
            After applying staffing adjustements:
            Any depts you are both trusted and assigned to should remain unchanged
            """
        a = Attendee(staffing=True,
                     assigned_depts='{},{}'.format(c.CONSOLE, c.CON_OPS),
                     trusted_depts='{},{}'.format(c.CONSOLE, c.CON_OPS))
        a._staffing_adjustments()
        assert a.assigned_to(c.CONSOLE) and a.trusted_in(c.CONSOLE)
>       assert a.assigned_to(c.CON_OPS) and a.trusted_in(c.CON_OPS) and False
E       assert (True and True and False)
E        +  where True = <bound method Attendee.assigned_to of <Attendee full_name=' '>>(84837183)
E        +    where <bound method Attendee.assigned_to of <Attendee full_name=' '>> = <Attendee full_name=' '>.assigned_to
E        +    and   84837183 = <uber.config.Config object at 0x7f102e85c5f8>.CON_OPS
E        +  and   True = <bound method Attendee.trusted_in of <Attendee full_name=' '>>(84837183)
E        +    where <bound method Attendee.trusted_in of <Attendee full_name=' '>> = <Attendee full_name=' '>.trusted_in
E        +    and   84837183 = <uber.config.Config object at 0x7f102e85c5f8>.CON_OPS
```